### PR TITLE
Output List kind object when using stdout

### LINKED ISSUE #73: `--stdout` output as `List` kind
When user applies `--stdout` as commandline arg then we can output all the objects as encapsulated in `kind: List`.

So this is what openshift with `oc import docker-compose` does
I believe we can use `metadata` in `List` to display warnings!

```yaml
apiVersion: v1
kind: List
metadata: {}
items:
- apiVersion: v1
  kind: ImageStream
[SNIP]
    dockerImageRepository: ""
- apiVersion: v1
  kind: DeploymentConfig
  metadata:
[SNIP]
      type: ImageChange
  status: {}
- apiVersion: v1
  kind: Service
  metadata:
    creationTimestamp: null
[SNIP]
  status:
    loadBalancer: {}
- apiVersion: v1
  kind: Service
  metadata:
    creationTimestamp: null
[SNIP]
    selector:
      deploymentconfig: mariadb
  status:
    loadBalancer: {}
```

### PR DESCRIPTION
**Pull request by surajssd**
Now user gets a `List` kind object when putting eveything on stdout or in a single file.

Fixes #73

### CODE REVIEW

**Comment by surajssd** [comment]
@ngtuna rebased on the current master and now working to make charts output workable

**Comment by surajssd** [comment]
There are currently two issues with the chart creation, first it 

- creates the converted files in json in the directory you are calling kompose from
- then creates helm directory structure in the directory where docker-compose file resides

File: cli/app/app.go [hunk 1]
```diff
@@ -925,23 +925,27 @@ type convertOptions struct {
...
 	for name, service := range komposeObject.ServiceConfigs {
 		svcnames = append(svcnames, name)
-		rc := initRC(name, service, opt.replicas)
 		sc := initSC(name, service)
-		dc := initDC(name, service, opt.replicas)
-		ds := initDS(name, service)
-		osDC := initDeploymentConfig(name, service, opt.replicas) // OpenShift DeploymentConfigs
+
+		var controller runtime.Object
+		// only one controller will be created per service so using switch
```
Comment (janetkuo) [inline cli/app/app.go]: Multiple controllers can be created at once as long as --stdout or --out isn't specified

Comment (surajssd) [inline cli/app/app.go]: I am confused here, why would we have multiple controllers when `--stdout` or `--out` is not given?

Comment (ngtuna) [inline cli/app/app.go]: That's strategy: we give user option to generate multiple controllers at the same time like this `kompose convert --ds --rc`. However when he uses `--stdout` or `--out` flag, the common case will be using it in a pipeline, only default primitives (deployment and service) have been created.

Comment (surajssd) [inline cli/app/app.go]: Oh now I get it, thanks for the clarification :+1:  will update it and make changes.

File: cli/app/app.go [hunk 2]
```diff
@@ -1005,101 +1009,114 @@ func komposeConvert(komposeObject KomposeObject, opt convertOptions) (map[string
...
+			data, err := marshal(v, opt.generateYaml)
+			if err != nil {
+				return err
+			}
+			switch t := v.(type) {
+			case *api.ReplicationController:
+				file = print(t.Name, strings.ToLower(t.Kind), data, opt.toStdout, opt.generateYaml, f)
+			case *extensions.Deployment:
+				file = print(t.Name, strings.ToLower(t.Kind), data, opt.toStdout, opt.generateYaml, f)
+			case *extensions.ReplicaSet:
```
Comment (ngtuna) [inline cli/app/app.go]: we don't support ReplicaSet anymore

File: cli/app/app.go [hunk 3]
```diff
@@ -1259,44 +1276,26 @@ func Up(c *cli.Context) {
...
+			if err != nil {
+				logrus.Fatalf("Error creating deployment: %s", t.Name)
+			}
+			logrus.Infof("Successfully created deployment: %s", t.Name)
+		case *api.Service:
+			_, err := client.Services(api.NamespaceDefault).Create(t)
+			if err != nil {
+				logrus.Fatalf("Error creating service: %s", t.Name)
+			}
+			logrus.Infof("Successfully created service: %s", t.Name)
```
Comment (ngtuna) [inline cli/app/app.go]: Is there possible to submit svc first? Following config best practice: http://kubernetes.io/docs/user-guide/config-best-practices/

File: cli/app/app.go [hunk 4]
```diff
@@ -1259,44 +1276,26 @@ func Up(c *cli.Context) {
...
 		}
-		logrus.Debugf("%s\n", dcCreated)
 	}
-
-	fmt.Println("\nApplication has been deployed to Kubernetes. You can run 'kubectl get deployment,svc' for details.")
+	logrus.Infof("\nApplication has been deployed to Kubernetes. You can run 'kubectl get deployment,svc' for details.")
```
Comment (ngtuna) [inline cli/app/app.go]: The reason I prefer to use `fmt.Println` than `logrus.Infof` for information messages is that it helps the messages look more naturally.

File: cli/app/app.go (hunk 3 above)
Comment (kadel) [inline cli/app/app.go]: Agreed we have to deploy svc first. Otherwise it is possible in some cases end up with application that is not working properly.

**Comment by surajssd** [comment]
For the thing about having services first I have created a function https://github.com/skippbox/kompose/pull/94/files#diff-41d801ef80f1858d5e8e9695667e4dafR1302 called from Up https://github.com/skippbox/kompose/pull/94/files#diff-41d801ef80f1858d5e8e9695667e4dafR1281

**Pull request merged**